rebar.config

```diff
@@ -229,13 +230,16 @@
                 ]},
                 {ct_opts, [{create_priv_dir, auto_per_tc}]}
             ]},
-            {eqc, [{project_app_dirs, ["eqc_test"]},
-                   {extra_src_dirs, [ "eqc/aecore_eqc"
-                                    , "eqc/aega_eqc"
-                                    , "eqc/aeminer_eqc"
-                                    , "eqc/aesophia_eqc"
-                                    , "eqc/aeutils_eqc"
-                                    ]}]}
+            {eqc, [ {project_app_dirs, ["eqc_test"]}
+                  , {extra_src_dirs, [ "eqc/aecore_eqc"
+                                     , "eqc/aega_eqc"
+                                     , "eqc/aeminer_eqc"
+                                     , "eqc/aechannel_eqc"
+                                     , "eqc/aesophia_eqc"
+                                     , "eqc/aeutils_eqc"
+                                     ]}
+                  , {overrides, [{override, [{erl_opts, [debug_info]}]}]}
```

Comment:
But this means that all other QuickCheck tests that do not use lager mocking are failing... that will give me a lot of emails from CI running the QuickCheck task every few hours...

Reply:
That is correct. I didn't notice that. So we have to configure 2 profiles. (1) `eqc` which works as before; (2) `eqc_lager_mock` which mocks lager.